### DISCUSSION TITLE
Could not add database error, when trying to add sqlcipher database

### DISCUSSION THREAD

**Discussion by omkar-ok** [pawelsalawa/sqlitestudio#4320] [Q&A]
i am new to sqlcipher
### Details
i have created a database named (test1.db) using sqlcipher terminal and added some data, database is all ok when i use it from terminal- commands, i can access this database. but when i tried to add / connecct to this test1.db from sqlitestudio it gives me error "Could not add database test1.db".

similarly, i have created one database test2.db using sqlitestudio and set it a password (key) as "test", and added some data, but when i tried to access this database from terminal sqlcipher and passed  ( PRAGMA key = "test"; ) i got error " file is enctypted or  not a database". 
please point me out where i am doing something wrong..

### SQLiteStudio version : 3.3.3
terminal sqlcipher version : 3.15.2

**Comment by pawelsalawa** [comment]
Try adding `PRAGMA cipher_compatibility = 3;` to the database configuration in SQLiteStudio (when you add the database to SQLiteStudio). It's that last field in the database dialog, where you can add custom statements to be executed at connecting.
